### ISSUE TITLE
Can you elaborate on how this is obsoleted by lispy?

### ISSUE THREAD

**Issue by Engelberg** [noctuid/parinfer-notes#1]
I just read through the lispy documentation, and I don't see any keystrokes that correspond to the functionality you describe here that would replace parinfer.

**Comment by noctuid** [comment]
I added all the hypothetical functionality mentioned here to lispy a while ago in the parinfer key theme, but the html documentation was just merged (see [here](http://oremacs.com/lispy/parinfer_index.html)). I'll update this repo to better explain things when I get the chance.

**Comment by Engelberg** [comment]
Thanks.  I was able to figure out how to get the lispy parinfer theme working, and it works pretty well.

Main problem seems to be using tab or shift-tab with regions, in which case really weird things happen, and undo doesn't actually undo them!  Also, I noticed backspace can sometimes get things into a strange state with unbalanced parens, and then there doesn't seem to be any good way to fix them.

But it was fun to play around with.  When it works, it feels really natural.
